tests/controllers/test_party_controller.py

```diff
@@ -0,0 +1,94 @@
+import unittest
+import mock
+from collections import namedtuple
+
+from response_operations_ui import create_app
+from response_operations_ui.controllers.party_controller import search_respondents, _get_search_respondents_url
+from response_operations_ui.exceptions.exceptions import SearchRespondentsException
+
```

Comment:
It worries me that `party_controller` didn't already have a test file.  This one only tests my code, but I'm adding a card to address this, as I think it's quite important.